.pre-commit-config.yaml

```diff
@@ -1,7 +1,7 @@
 exclude: '^docs/conf.py'
 
 repos:
-- repo: git://github.com/pre-commit/pre-commit-hooks
+- repo: https://github.com/pre-commit/pre-commit-hooks
```

Comment:
I had to change this otherwise pre-commit would just hang.

Reply:
I do, which is why this surprised me as well. It was almost as if github was being blocked but only through this mechanism (I used `git@` when cloning this repo for example). Let me see if I can snag the error I was seeing in the log files.

Reply:
Hangs here:
```
pre-commit run --all-files
[INFO] Initializing environment for git://github.com/pre-commit/pre-commit-hooks.

```
and then after a while fails with this:
```
An unexpected error has occurred: CalledProcessError: command: ('/usr/bin/git', 'fetch', 'origin', '--tags')
return code: 128
expected return code: 0
stdout: (none)
stderr:
    fatal: unable to connect to github.com:
    github.com[0: 140.82.113.4]: errno=Operation timed out


```